trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/chandler/CCharacterConstant.java

```diff
@@ -44,6 +44,7 @@ public class CCharacterConstant {
 	/**
 	 * Representation of this character literal in source code.
 	 */
+	@SuppressWarnings("unused")

```

Comment:
Why did you add these `@SuppressWarnings` (also in other files)? You did not actually change this class, right? If this is unused indeed, it should be rather removed than suppressed, but also not in this PR.